.bit/responses/1.8-Week 1 Step 8.md

```diff
@@ -11,6 +11,7 @@ name: Week 1 Step 8
 
 ### ✅  Task:
 - [ ] Add parameters to the request to name your cat picture with the result from nameFinder
+- [ ] To move on, place your function URL in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) and comment the name of the secret below so we can check your function. Remember to commit the function's code in a file named `CatAPI.js` to the root of the `week1` branch!
```

Comment:
Make the branch part and committing to a file separate tasks. This is just to improve readability